update readme (#2)
DevExpress-Examples/xaf-how-to-enforce-password-complexity-in-xaf: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2849)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# XAF - How to enforce password complexity

This example shows how to validate passwords in XAF applications.

![image](https://github.com/DevExpress-Examples/XAF_how-to-enforce-password-complexity-in-xaf-e2849/assets/14300209/522c4ee7-66b1-4579-a7f1-bd012a20237d)

## Implementation Details

> The easiest way to do this is to use [Model Editor](https://docs.devexpress.com/eXpressAppFramework/112582/concepts/application-model/model-editor) to create a special validation rule that checks all new passwords against a regex expression. 

Reviewer: ```suggestion
We recommend that you use [Model Editor](https://docs.devexpress.com/eXpressAppFramework/112582/concepts/application-model/model-editor) to create a special validation rule that checks all new passwords against a regex expression. 
```
